### DISCUSSION TITLE
🔗 🚀 Feature Request: Taxonomy Custom Query from CPT (Duplicate of #21372)

### DISCUSSION THREAD

**Discussion by guga06cl** [elementor/elementor#22592] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Currently, Elementor Pro does not support querying and displaying data from custom taxonomies created with Advanced Custom Fields Pro (ACF Pro). This limitation restricts the customization and flexibility capabilities of Elementor Pro, particularly for developers who frequently use ACF Pro to create Custom Post Types (CPTs) and their associated taxonomies.

### Describe the solution you'd like

I propose that Elementor Pro add functionality to query and display data from custom taxonomies created with ACF Pro. This feature would allow developers to fully leverage the dynamic capabilities of both Elementor Pro and ACF Pro, thereby expanding the customization potential of websites developed using these tools.

### Describe alternatives you've considered

One alternative solution would be to use WooCommerce to create a catalogue-like website. However, this option often involves unnecessary extra code that can potentially affect the site's performance. Furthermore, not all websites require the eCommerce capabilities that WooCommerce offers, rendering much of its functionality redundant in these cases.

Another alternative would be to migrate to other pagebuilders that do have this feature, but this is not the idea, obviously.

### Additional context

The ability to query and display data from custom taxonomies created with ACF Pro would allow for the development of lighter, cleaner code when creating catalogue-like websites using CPTs. For instance, it could enable the emulation of the "Shop" page of WooCommerce through CPTs without the need for the additional code that comes with WooCommerce.

As of now, due to the lack of this feature in Elementor Pro, it's impossible to achieve this, limiting the possibilities for non-eCommerce catalogue sites or pushing to users to buy 3rd party plugins like Crocoblock, that are able to do it, but:

1. Costs is increased
2. Unnecesary extra code is added

**Comment by nicholaszein** [comment]
Hello there, @guga06cl!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #21372

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards
